apps/src/sites/studio/pages/lessons/student_lesson_plan.js

```diff
@@ -37,7 +37,9 @@ async function displayLessonOverview() {
           announcement.details,
           announcement.link,
           announcement.type,
-          announcement.visibility
+          announcement.visibility,
+          announcement.dismissible,
+          announcement.buttonText
```

Comment:
this is quite a lot of positional arguments. you could reduce the risk of error here by using object destructuring in the method definition:
```
export const addAnnouncement = ({
  notice,
  details,
  link,
  type,
  visibility,
  dismissible,
  buttonText
}) => ({
```

```
store.dispatch(addAnnouncement(announcement))
```

Reply:
Oh cool! Ya I'll do that